runtime/doc/options.txt

```diff
@@ -3267,7 +3267,7 @@ A jump table for the options with a short description can be found at |Q_op|.
 				     >:SignColumn,B:SpellBad,P:SpellCap,
 				     R:SpellRare,L:SpellLocal,-:Conceal,
 				     +:Pmenu,=:PmenuSel,x:PmenuSbar,
-				     X:PmenuThumb")
+				     X:PmenuThumb,q:QuickFixLine")
```

Comment:
Indeed, we cannot "just remove" &highlight as it is what currently defines ui highlights. We could consider making it read-only or replace it with a builtin table, but that would be another issue/PR...